src/lib/UtapiClient.js

```diff
@@ -435,14 +445,14 @@ export default class UtapiClient {
                     callback);
             }
             // number of objects counters
-            let index;
             let actionErr;
             let actionCounter;
             let key;
-            const paramsArrLen = paramsArr.length;
             const cmds2 = [];
             const noErr = paramsArr.every((p, i) => {
-                index = i * paramsArrLen;
+                // We want the first element of every group of two commands
+                // returned from ioredis.
+                const index = i * commandsGroupSize;
```

Comment:
Update here since if `paramsArrLen` is greater than 2 (which is the case when adding the user level), we will be off by one.